### PR TITLE
Add typescript typings

### PR DESCRIPTION
**Pull request by nchen63**

### CODE REVIEW

File: src/pinch-zoom.d.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    // deprecated events
+    zoomStartEventName?: string;
+    zoomUpdateEventName?: string;
+    zoomEndEventName?: string;
+    dragStartEventName?: string;
+    dragUpdateEventName?: string;
+    dragEndEventName?: string;
+    doubleTapEventName?: string;
+    verticalPadding?: number;
+    horizontalPadding?: number;
```
Comment (sandstrom) [inline src/pinch-zoom.d.ts]: This isn't a deprecated event

File: src/pinch-zoom.d.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    zoomOutFactor?: number;
+    animationDuration?: number;
+    maxZoom?: number;
+    minZoom?: number;
+    draggableUnzoomed?: boolean;
+    lockDragAxis?: boolean;
+    setOffsetsOnce?: boolean;
+    use2d?: boolean;
+
+    // events
```
Comment (sandstrom) [inline src/pinch-zoom.d.ts]: these are callback methods, not events

File: src/pinch-zoom.d.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+type PinchZoomEventHandler = (target: PinchZoom, event: TouchEvent) => void;
+
+interface IPinchZoomOptions {
```
Comment (sandstrom) [inline src/pinch-zoom.d.ts]: Why is this one named `IPinchZoomOptions` and not `PinchZoomOptions`?

Comment (librajt) [reply src/pinch-zoom.d.ts]: I think `I-prefix` refers to type `Interface`

Comment (sandstrom) [reply src/pinch-zoom.d.ts]: Alright! 😄 

Happy to hear our thoughts about my other questions above.

This is a nice addition, so would love to get it merged.

Comment (nchen63) [reply src/pinch-zoom.d.ts]: I removed the comments, which should address those issues

**Comment by sandstrom** [comment]
Thanks @nchen63! 🏅

**Pull request merged**